README.md

```diff
@@ -1,143 +1,126 @@
+[Docker Desktop](https://www.docker.com/products/docker-desktop), which will install both Docker and Docker Compose as well as Docker
+Machine, which is required to run Docker on Mac/Windows hosts.
 
-<!-- IDEA: For brevity, consider moving steps 3 and 4 to a different section or document -->
 
-### For Isolated Instance Like Production
+### Code Configuration
 
-Skip further configuration; you may use the default configuration.
+After you clone the repository locally, there are several configuration steps required to prepare the project.
 
-### For Isolated Containerized Local Development
+#### Required
 
-1. Run any `docker-compose` command with `docker-compose.dev.yml` configuration, ex:
+1. Create a `.env` file\* at the root of the project with this content:
 
     ```bash
-    docker-compose -f docker-compose.dev.yml …
+    CUSTOM_ASSET_DIR=example-cms
     ```
 
-### (Optional) Custom Configuration
+    Where `example-cms` is the project to be run. See [Custom Resources per CMS Project](#custom-resources-per-cms-project).
 
-Configuration is stored in `default_secrets.py` and may be customized by creating a `secrets.py`.
+1. Initialize / Update submodules.
+    1. `git submodule init`\
+        Adds `cms-site-resources` repo as submodule at `taccsite_custom/`. Only necessary once per `CORE-cms` repo clone.
+    2. `git submodule update`\
+        Downloads code from pinned commit of `cms-site-resources` repo to `taccsite_custom/`.
 
-1. Copy `taccsite_cms/default_secrets.py` as new `taccsite_cms/secrets.py` file.
-2. Update the `secrets.py` file.
+\* A ["dotenv" file](https://hexdocs.pm/dotenvy/dotenv-file-format.html) is a file, with or without a name, that has the `.env` extension.
 
-### (Optional) Custom Resources per CMS Project
+#### Dependent
 
-All CMS projects (besides the stand-alone CMS core), store project-specific resources in the `taccsite_custom` submodule.
+##### If You Run this CMS Independent of [Core-Portal](https://github.com/TACC/Core-Portal/)
 
-1. (For Production) Copy `docker-compose.yml` as new `docker-compose.custom.yml` file, and in the new file:
-    - To `cms`:`volumes` list, add an entry `/code/taccsite_custom/name-of-project/static`.*
-2. Create and update `secrets.py`. _See [Custom Configuration](#optional-custom-configuration)._
-    - Setup existing CMS project by manually appending secrets from `taccsite_custom/name-of-project/secrets.py`.*
-    - For new CMS projects, add custom and unique resources and configuration to `taccsite_custom/name-of-project/`.*
-3. Update the `.env` at the root of the project, with the content `CUSTOM_ASSET_DIR=name-of-project`.*
-4. Re-build static assets, so that project-specific assets are built. _See ["Static Files"](/README.md#static-files)._
+Add `INCLUDES_CORE_PORTAL = False` to `taccsite_cms/settings_local.py` (to avoid [Not Found: `core/markup/nav/`](https://github.com/TACC/Core-CMS/wiki/Not-Found%3A--core-markup-nav)).
```

Comment:
For this section, I was a little confused that a `taccsite_cms/settings_local.py` file didn't already exist, and the table below helped clarifying that such files can contain only necessary flags to change settings. Consider moving the table + it's caption to the top of the '**Dependent**' section, as it could inform setting up either with or without local Core-Portal instance.

Reply:
Excellent feedback, thank you. Task added.

Reply:
Yes, that section reads better now! And the other commits look good too.